commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -297,7 +297,7 @@
                 } else {
                     $scope.rollback = false;
                 }
-            } else if (newValue !== undefined && newValue.name === "") {
+            } else if (newValue && newValue.name === "") {
```

Comment:
@pgesek we are checking whether newValue.name is empty. So, shouldn't it be (newValue && !newValue.name)?